sig/helpers.rbs

```diff
@@ -0,0 +1,35 @@
+module Helpers
+  # Convert an Hash to json
+  #
+  def to_json: (String|Hash[Symbol, String] body) -> String
+
+  # Converts each key of a hash to symbols
+  #
+  def symbolize_hash: (Hash[Symbol|String, String] hash) -> Hash[Symbol, String]
```

Comment:
Do we always force the `Hash[Symbol|String, String]` type? I think it's safer if we opt for `Hash[Symbol|String, untyped]`, so that we can convert nested hashes as well, for example. Wdyt?

Reply:
indeed we're not sure what's the second parameter will be